ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx

```diff
@@ -93,6 +93,9 @@ const Header = ({
           {mediaCollections && (
             <LiveHeaderMedia
               mediaCollection={mediaCollections}
+              eventTrackingData={{
+                componentName: 'live-header-media',
+              }}
```

Comment:
Should we pass in the eventTracking here, or should the component itself always have event tracking enabled? (Not sure what we do across other components). In that case, we would move this to within the LiveHeaderMedia component.

Reply:
<img src="https://github.com/user-attachments/assets/d060342a-e495-4f14-9ca2-1d515f75c331" width="320">